content/zh/blog/2021/better-external-authz/index.md

```diff
@@ -3,11 +3,11 @@ title: 更好的外部授权方式集成
 subtitle: 使用 AuthorizationPolicy 将外部授权系统（例如 OPA、oauth2-proxy 等）与 Istio 进行集成
 description: AuthorizationPolicy 现在支持以 CUSTOM 自定义方式委托外部系统进行授权操作。
 publishdate: 2021-02-09
-attribution: Yangmin Zhu (Google)
+attribution: Yangmin Zhu (Google); Translated by Wilson Wu (DaoCloud)
```

Comment:
This change to attribution shall be completed in the first localization. An update to this attribution may cause question to check.

I have verified the [history](https://github.com/istio/istio.io/commits/master/content/zh/blog/2021/better-external-authz/index.md) and confirmed that both the author and maintainer listed are Wilson, no any other person involved. Everything appears to be correct.

Reply:
Thanks, it is myself too.